requirements.txt

```diff
@@ -0,0 +1,5 @@
+certifi==2020.6.20
```

Comment:
my WINDOWS was recently installed and used only to run the miningframework and when I created requirements.txt automatically all these dependencies appeared.  I believe that all these dependencies have been installed together with "requests".  Leave it this way because it is exactly the state that is working on my machine.